### DISCUSSION TITLE
F1 Testing

### DISCUSSION THREAD

**Discussion by telemetryf1** [theOehrly/Fast-F1#314] [Q&A]
Hi sorry, which is the parameter to insert into get_session() function to get testing laptimes and graphs? I mean, if for Monza is 'Monza', which is for Bahrein testing? Thanks at all

**Comment by theOehrly** [comment]
You need to use ``get_testing_session``. See https://theoehrly.github.io/Fast-F1/fastf1.html#fastf1.get_testing_session
